app/src/main/java/de/danoeh/antennapod/fragment/AddFeedFragment.java

```diff
@@ -197,7 +197,7 @@ private void addLocalFolderResult(final Uri uri) {
 
     private Feed addLocalFolder(Uri uri) {
         getActivity().getContentResolver()
-                .takePersistableUriPermission(uri, Intent.FLAG_GRANT_READ_URI_PERMISSION);
+                .takePersistableUriPermission(uri, Intent.FLAG_GRANT_WRITE_URI_PERMISSION);
```

Comment:
Shouldn't this take both, read and write?

Reply:
I think write implies read, but will add both just in case.